modules/claims_api/spec/support/pdf_matcher.rb

```diff
@@ -4,7 +4,16 @@
 
 RSpec::Matchers.define :match_pdf_content_of do |expected|
   match do |actual|
-    PDF::Inspector::Text.analyze(actual).strings == PDF::Inspector::Text.analyze(expected).strings
+    actual_reader = PDF::Reader.new(actual)
+    actual_pages = actual_reader.pages.size
+    actual_text = actual_reader.pages.map(&:text)
+
+    expected_reader = PDF::Reader.new(expected)
+    expected_pages = expected_reader.pages.size
+    expected_text = expected_reader.pages.map(&:text)
+
+    # Check both page count and text content in case there are extraneous blank pages
+    actual_pages == expected_pages && actual_text == expected_text
```

Comment:
blatantly stolen from 
https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/spec/support/pdf_matcher.rb#L16
to resolve the error i was seeing: "PDF does not contain EOF marker"

might be valuable to move that helper file outside modules for all to share but didn't for now considering how much is already changing with this pr.